### PR TITLE
Add note on using PL dynos for scripts

### PR DESCRIPTION
**Pull request by gaganawhad**

### CODE REVIEW

File: best-practices/README.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ Scripts
 -------
 
 * Write idempotent scripts. This is helpful when re-running scripts that have failed midway.
+* Use PL dynos when running a script on production. They help speed up running time appreciably. `--size=performance-l`
```
Comment (benhutton) [inline best-practices/README.md]: @gaganawhad did you use PL or PM dynos?

Comment (gaganawhad) [inline best-practices/README.md]: @benhutton I used PM, but I thought if we are making a note, we should probably use the ones that are most performant. What do you think?

Comment (benhutton) [inline best-practices/README.md]: @gaganawhad I would go with PM. There's probably not too much difference between PM and PL, and if you are being told that you can use PM, then you'll be also thinking about potentially using PL.

ALSO, I would give this as advice, not directive. Often, 1x is just fine. Maybe change it to: `Use PM or PL dynos when running long scripts on production.`

Comment (gaganawhad) [inline best-practices/README.md]: @benhutton I pushed an update.

**Pull request merged**